docs/index.md

```diff
@@ -23,7 +23,15 @@
 
 > A rising tide lifts all boats. -- United States President, John F. Kennedy (borrowed from the New England Council)
 
-Inspired by the proverb _“A rising tide lifts all boats”_, when we lower the barrier of entry to writing and choosing quality code for enough people, it will lift the quality of code across the whole WordPress ecosystem. Tide’s vision is to make it easy to improve the quality of code throughout the WordPress ecosystem and help WordPress site owners make better choices about plugins and themes.
+Inspired by the proverb _“A rising tide lifts all boats”_, we want to lift the quality of code across the whole WordPress ecosystem. Tide’s vision is to run automated quality testing for all WordPress plugins and themes and make those test results visible for both the authors and to end users of those plugins and themes. This will help improve the quality of code throughout the WordPress ecosystem and help WordPress site owners make better choices about plugins and themes.
```

Comment:
```suggestion
Inspired by the proverb _“A rising tide lifts all boats”_, we want to lift the quality of code across the whole WordPress ecosystem. Tide’s vision is to run automated quality testing for all WordPress plugins and themes and make those test results visible for both the authors and the end users of those plugins and themes. This will help improve the quality of code throughout the WordPress ecosystem and help WordPress site owners make better choices about plugins and themes.
```